packages/contracts-bedrock/src/L2/L2CrossDomainMessenger.sol

```diff
@@ -66,13 +70,34 @@ contract L2CrossDomainMessenger is CrossDomainMessenger, ISemver {
         override
         returns (bool)
     {
+        // TODO: Remove this comment 5_500 accounted for in this call.
+        // Accounting for: Cold Sload (2100) + Cold Static (2600) = 4_700 + 800 (buffer extra logic)
         address l2MessageValidator = L1Block(Predeploys.L1_BLOCK_ATTRIBUTES).l2MessageValidator();
+
         if (l2MessageValidator == address(0)) {
             return true;
         }
-        return IL2MessageValidator(l2MessageValidator).validateMessage(
-            _nonce, _sender, _target, _value, _minGasLimit, _message
+
+        bytes memory callData = abi.encodeWithSelector(
```

Comment:
this could cost non-neglible gas right?

Reply:
That's what the 800 is accounting for, but I agree we need to figure out the exact cost